### PR TITLE
Update results for Russian models

### PR DESCRIPTION
**Pull request by artemsnegirev**
This PR updates results for several models to update Russian leaderboard (https://github.com/embeddings-benchmark/leaderboard/pull/26).

Models are updated:
- BAAI__bge-m3                       
- ai-forever__sbert_large_nlu_ru  
- intfloat__e5-mistral-7b-instruct 
- intfloat__multilingual-e5-large          
- intfloat__multilingual-e5-small
- ai-forever__sbert_large_mt_nlu_ru  
- cointegrated__rubert-tiny2      
- intfloat__multilingual-e5-base    
- intfloat__multilingual-e5-large-instruct

Most part of update brings minor changes but now uses one version (1.14.12) and kg_co2_emissions is computed (that wasn't). Results for multilingual MassiveIntent, MassiveScenario and STS22 are not changed. And instruct models now use detailed instructions from https://github.com/embeddings-benchmark/mteb/pull/1163.

### CODE REVIEW

**Comment by Samoed** [comment]
@KennethEnevoldsen @Muennighoff Can you merge this?

File: results/intfloat__e5-mistral-7b-instruct/07163b72af1488142a360786df853f237b1a3ca1/KinopoiskClassification.json
Comment (KennethEnevoldsen) [inline results/intfloat__e5-mistral-7b-instruct/07163b72af1488142a360786df853f237b1a3ca1/KinopoiskClassification.json:1]: I assume the results change due to the prompt?

File: results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json
Comment (KennethEnevoldsen) [inline results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: quite a large change in accuracy here, why is that?

**Review by KennethEnevoldsen — COMMENTED** [review]
Thanks for the ping! Looked at a few samples probably worth discussing these before merging.

File: results/intfloat__e5-mistral-7b-instruct/07163b72af1488142a360786df853f237b1a3ca1/KinopoiskClassification.json
Comment (Samoed) [reply results/intfloat__e5-mistral-7b-instruct/07163b72af1488142a360786df853f237b1a3ca1/KinopoiskClassification.json:1]: Yes, I previously run without prompts

File: results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json
Comment (artemsnegirev) [reply results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: I was able to reproduce the results for 1.12.75, so I guess due to some changes between this versions (1.12.75 -> 1.14.12). And it's not only for me5-small and this task (Georeview) but for almost all models and Classification/Clustering tasks.

Maybe you have any hypothesis what changes could affect results?

Comment (artemsnegirev) [reply results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: Found that running MultiLabelClassification tasks at first cause the problem. Code to reproduce:

```bash
# mteb==1.14.12

MODEL_NAME="cointegrated/rubert-tiny2"

mteb run \
    -m $MODEL_NAME -l rus --output_folder results \
    --co2_tracker true --verbosity 2 --batch_size 16 \
    -t \
       "SensitiveTopicsClassification" \
       "GeoreviewClassification"
```

GeoreviewClassification is `0.408935546875` while single run gives `0.39638671875`. Also checked Retrieval and Reranking tasks and they are ok.

Comment (KennethEnevoldsen) [reply results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: So running the MultiLabelClassification first changes the result? It seems to be an issue with a seed being manipulated

Comment (artemsnegirev) [reply results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: Yes, I will run this tasks separately and update the results then

Comment (KennethEnevoldsen) [reply results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: I will look into this in a PR as well - see if we can get it fixed.

Comment (KennethEnevoldsen) [reply results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: More generally we should probably try to fix this issue [here](https://github.com/embeddings-benchmark/mteb/issues/942).

Comment (KennethEnevoldsen) [reply results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: PR here: https://github.com/embeddings-benchmark/mteb/pull/1193

**Comment by artemsnegirev** [comment]
@KennethEnevoldsen I've updated affected tasks. It seems Clustering also has to be run separately. Also found clustering results for sbert/rubert models in v1.12.25 gives better results and I reproduced this difference.

File: results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json
Comment (Samoed) [reply results/intfloat__multilingual-e5-small/e4ce9877abf3edfe10b0d82785e83bdcb973e22e/GeoreviewClassification.json:1]: My results for bge-m3 for comparison 
[new_results.tar.gz](https://github.com/user-attachments/files/16851962/new_results.tar.gz)

**Comment by KennethEnevoldsen** [comment]
> @KennethEnevoldsen I've updated affected tasks. It seems Clustering also has to be run separately. Also found clustering results for sbert/rubert models in v1.12.25 gives better results and I reproduced this difference.

Right so order also matters here. Well, we knew it was a problem given [this](https://github.com/embeddings-benchmark/mteb/issues/942), but we should def. get that patched up (though it might be a major version bump)

**Comment by artemsnegirev** [comment]
@KennethEnevoldsen

> Well, we knew it was a problem given https://github.com/embeddings-benchmark/mteb/issues/942, but we should def. get that patched up (though it might be a major version bump)

We've made several experiments (@Samoed) with the bge-m3 model and could not achieve the same results. As mentioned in https://github.com/embeddings-benchmark/mteb/issues/942, this is not a big problem. For now we would like to go with these results and will update them in new versions. Waiting #25 to be merged.

**Comment by KennethEnevoldsen** [comment]
That works for me as well

**Comment by Samoed** [comment]
@KennethEnevoldsen I think this PR can be merged. After that, I will update the paths in my branch (or whatever else we decide)

**Pull request merged**